ui/components/connections/styles.js

```diff
@@ -242,6 +244,168 @@ const styles = (theme) => ({
   },
 });
 
+export const ChipWrapper = styled(Chip)({
+  width: '13rem',
+  maxWidth: '13rem',
+  minWidth: '9rem',
+  textAlign: 'left',
+  cursor: 'pointer',
+  '& .MuiChip-label': {
+    flexGrow: 1,
+  },
+  overflow: 'hidden',
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  border: '1px solid rgba(255, 255, 255, 0.23)',
+});
+
+export const StatusChip = styled(Chip)({
+  minWidth: '142px !important',
+  maxWidth: 'max-content !important',
+  display: 'flex !important',
+  justifyContent: 'flex-start !important',
+  borderRadius: '3px !important',
+  padding: '6px 8px',
+  '& .MuiChip-label': {
+    paddingTop: '3px',
+    fontWeight: '400',
+  },
+  '& .MuiSvgIcon-root': {
+    marginLeft: '0px !important',
+  },
+  '&:hover': {
+    boxShadow: '0px 1px 2px 0px rgba(0, 0, 0, 0.25)',
+  },
+});
+
+export const DiscoveredChip = styled(Chip)(() => ({
+  '& .MuiChip-label': {
+    color: notificationColors.info,
+  },
+  background: `${notificationColors.info}30 !important`,
+  '& .MuiSvgIcon-root': {
+    color: `${notificationColors.info} !important`,
+  },
+}));
+
+export const DeletedChip = styled(Chip)(({ theme }) => ({
+  '& .MuiChip-label': {
+    color: theme.palette.background.error.default,
+  },
+  background: `${theme.palette.background.error.default}30 !important`,
+  '& .MuiSvgIcon-root': {
+    color: `${theme.palette.background.error.default} !important`,
+  },
+}));
+
+export const RegisteredChip = styled(Chip)(({ theme }) => ({
+  '& .MuiChip-label': {
+    color: theme.palette.mode === 'dark' ? '#86B2C6' : '#477E96',
+  },
+  background: `${theme.palette.mode === 'dark' ? '#86B2C6' : '#477E96'}30 !important`,
+  '& .MuiSvgIcon-root': {
+    color: `${theme.palette.mode === 'dark' ? '#86B2C6' : '#477E96'} !important`,
+  },
+}));
+
+export const ConnectedChip = styled(Chip)(({ theme }) => ({
+  '& .MuiChip-label': {
+    color: theme.palette.background.success.default,
+  },
+  background: `${theme.palette.background.success.default}30 !important`,
+  '& .MuiSvgIcon-root': {
+    color: `${theme.palette.background.success.default} !important`,
+  },
+}));
+
+export const IgnoredChip = styled(Chip)(({ theme }) => ({
+  '& .MuiChip-label': {
+    color: theme.palette.mode === 'dark' ? '#9FAFB6' : '#51636B',
```

Comment:
currently these particular state color are not available in sistent that's why kept as hard code names. Will later on add it to sistent so we can use it as tokens.